docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM datascience-registry.cn-beijing.cr.aliyuncs.com/tensorflow/tensorflow:1.15.5
+
+COPY docker/sources_18.04.list /etc/apt/sources.list
+
+RUN cat /etc/apt/sources.list
```

Comment:
remove

Reply:
it is necessary to include this for later apt commands to take effect, the reason is not known